pkgs/applications/audio/espeak-ng/default.nix

```diff
@@ -1,5 +1,6 @@
 { stdenv, lib, fetchFromGitHub, autoconf, automake, which, libtool, pkg-config
-, ronn
+, ronn, substituteAll
+, mbrolaSupport ? true, mbrola
```

Comment:
The voices are pretty big. Maybe we should disable it by default.

Reply:
The improvement is very noticeable and it’s not like TTS is enabled anywhere by default. I would just leave it in for now and focus on writing a patch for espeak for finding mbrola voices in `XDG_DATA_DIRS` when I have some time.

Reply:
@jtojnar yes, that would be perfect. then one can install just the language they need